crates/kitsune_p2p/types/src/config.rs

```diff
@@ -21,7 +21,7 @@ pub mod tuning_params_struct {
     use std::collections::HashMap;
 
     macro_rules! mk_tune {
-        ($($(#[doc = $doc:expr])* $i:ident: $t:ty = $d:expr,)*) => {
+        ($($(#[doc = $doc:expr])* $(#[cfg($cfg:meta)])? $i:ident: $t:ty = $d:expr, $(#[cfg_not($cfg_alt:meta)] $i_alt:ident: $t_alt:ty = $d_alt:expr,)?)*) => {
```

Comment:
This is one hard to read line - hallelujah!

Reply:
My brain melted multiple times trying to write it ;)